### PR TITLE
Create new IA for observability docs

### PR DESCRIPTION
**Pull request by dedemorton**
Closes https://github.com/elastic/obs-docs-projects/issues/247

Preview link: https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/index.html

## Top-level TOC changes

image

## Questions to consider during review

- Is the latest nav design (without verbs or gerunds) effective or too slimmed down? Note that the page titles provide a lot more context for search engines to scoop up.
- Is the topic nesting excessive or does it make the navigation easier to scan and use?
- Should universal profiling live under infrastructure instead of applications?
- Should Kubernetes live under Infrastructure monitoring or as it's own top-level container, maybe at the same level as a container called "Hosts"?
- Are we OK with proceeding with an Information Architecture that exposes our gaps and promotes topics that we know are dated (cloud monitoring, for example)?
- Are we OK with having a set of top-level topics that are not parallel in nature? For example, some top-level topics (like the topic about monitoring applications and services) is simply a jump table that points to other content. Other topics are meaty with all the details in a single topic. Given the limited content we currently have for some topics, this problem is unavoidable.

## TODO
- [x] Add the "quickstarts" under "start here" (after https://github.com/elastic/observability-docs/pull/4168 is merged). We may want to flatten out the nav since "quickstart" will be redundant with "start here"). I like this idea, but we need to find out if the info that introduces the quickstarts is necessary. We could add this info in the box at the top or something, but it might be a bit repetitive and distracting.
- [x] Add content to placeholder topics:
  - [x] start-here.asciidoc
  - [x] application-and-service-monitoring.asciidoc
  - [x] cloud-monitoring.asciidoc
  - [x] incident-management.asciidoc
- [x] Check for links that point to topics where we've added new abbreviated titles and make sure the link text looks OK.

TODO after merging:

- [ ] Set up redirects for any renamed files or anchor IDs:
  - [ ] Redirect analyze-metrics.html to monitor-infrastructure-and-hosts
  - [ ] Redirect quickstarts-overview.html to observability-get-started.html
  - [ ] Look at redirects.asciidoc for other possible redirects to set up.
- [ ] Resolve TODOs (inline in source files) related to this effort -- these are mostly related to adding content or deleting old files.
- [ ] Make troubleshooting titles consistent across obs docs (to gerund or not to gerund, that is the question)
- [ ] Apply similar changes to the serverless docs.

### CODE REVIEW

File: docs/en/observability/apm.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/apm.asciidoc:5]: It seems strange to use the acronym "APM" when the full text doesn't include the "M" word.

File: docs/en/observability/application-and-service-monitoring.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/application-and-service-monitoring.asciidoc:16]: This feels strange to me since (1) it's the only language-specific item at this level and (2) we also include content related to Java applications throughout the APM section. That said, I don't have an alternative to suggest so 🤷 .

File: docs/en/observability/ci-cd-observability.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/ci-cd-observability.asciidoc:2]: Should we include "Continuous Integration and Continuous Delivery (CI/CD)" in the title? It's a very common term so we probably don't need to, but I'm not sure where we draw the line on defining acronyms on the first mention. We _ spell out "Amazon Web Services (AWS)" in the title of that page. 🤷

File: docs/en/observability/monitor-gcp.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/monitor-gcp.asciidoc:2]: Maybe add "(GCP)" to the end so both the full name and the acronym are in the title, which might have an influence on SEO.

File: docs/en/observability/synthetics-intro.asciidoc
Comment (colleenmcginnis) [inline docs/en/observability/synthetics-intro.asciidoc:5]: I prefer "Synthetic monitoring", but I don't have a specific reason.

**Review by colleenmcginnis — COMMENTED** [review]
Lookin' good ✨

File: docs/en/observability/application-and-service-monitoring.asciidoc
Comment (dedemorton) [reply docs/en/observability/application-and-service-monitoring.asciidoc:16]: Yah I feel the same. It's really an outlier. I wonder if we event want to continue publishing it since the info is so old. Maybe one option is to hide it from the TOC. I don't have a good answer to this problem.

File: docs/en/observability/apm.asciidoc
Comment (dedemorton) [reply docs/en/observability/apm.asciidoc:5]: @colleenmcginnis Do you think it's better to just remove APM or spell it all out? I'm trying to keep the nav titles relatively short, if possible.

File: docs/en/observability/ci-cd-observability.asciidoc
Comment (dedemorton) [reply docs/en/observability/ci-cd-observability.asciidoc:2]: I think it wouldn't hurt and might help with SEO

**Comment by alaudazzi** [comment]
@dedemorton thank you for leading this doc refactoring!
A few comments/suggestions on the overall structure:

- The [Start here](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/observability-start-here.html) page is a nice entry point. How about making it lighter by removing the various “Learn about, learn how, learn more” with something like this:
_New to Elastic Observability? Discover more about our Observability features and how to get started._

- The vicinity of Quick starts and Get started is confusing. How about promoting the Quick starts one level up and make them more visible in the TOC? 

- In [Applications and services](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/application-and-service-monitoring.html), it is weird to have “monitoring” for most of the TOC entries, except for APM and Synthetics.

- I love the [Cloud](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/cloud-monitoring.html) section!

- In [Monitor AWS with Elastic Agent](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/monitor-aws-elastic-agent.html) and [Monitor AWS with Amazon Data Firehose](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/ingest-aws-firehose.html), we can alleviate the TOC by removing Monitor from headers?
    image

- [Reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/reference.html) seems vague and is not adding much value to the navigation. How about moving [Fields reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/fields-reference.html) and [Elastic Entity Model](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/elastic-entity-model.html) on level up and removing [Reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/reference.html)?

Now to your questions:
1. Is the latest nav design (without verbs or gerunds) effective or too slimmed down? Note that the page titles provide a lot more context for search engines to scoop up.
  Looks OK.  Otherwise, a possible alternative could be:
Start here
[Applications and services](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/application-and-service-monitoring.html)
[Cloud](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/cloud-monitoring.html)
[Infrastructure and hosts](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/monitor-infrastructure-and-hosts.html)
Logs
[CI/CD](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/ci-cd-observability.html)
Kubernetes
[Observability AI Assistant](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/obs-ai-assistant.html)
[Incident management](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/incident-management.html)
[Fields reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/fields-reference.html)
[Elastic Entity Model](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/elastic-entity-model.html)

1. Is the topic nesting excessive or does it make the navigation easier to scan and use?
It goes down to level 4, but it seems to be unavoidable considering how the TOC is recompacted

1. Should universal profiling live under infrastructure instead of applications?
I don’t know

1. Should Kubernetes live under Infrastructure monitoring or as it's own top-level container, maybe at the same level as a container called "Hosts"?
I would leave it there 

1. Should the getting started guides live under "start here" or within relevant sections?
Having them consolidated in one single place makes them more usable. 

1. Are we OK with proceeding with an Information Architecture that exposes our gaps and promotes topics that we know are dated (cloud monitoring, for example)?
Upper management recently claimed that AWS and Azure were not discoverable enough. This reorg exposes them as they should.

1. Are we OK with having a set of top-level topics that are not parallel in nature? For example, some top-level topics (like the topic about monitoring applications and services) is simply a jump table that points to other content. Other topics are meaty with all the details in a single topic. Given the limited content we currently have for some topics, this problem is unavoidable.
If it’s unavoidable, then it’s OK :-)

**Review by alaudazzi — COMMENTED** [review]
Great rework! I've left a few comments/suggestions.

File: docs/en/observability/apm.asciidoc
Comment (colleenmcginnis) [reply docs/en/observability/apm.asciidoc:5]: I'm inclined to say we should spell it out because I think "Application performance monitoring" as a phrase is more universally recognizable. I'm curious what @bmorelli25 thinks.

**Comment by dedemorton** [comment]
@alaudazzi Thanks for your input! Just to clarify, I didn't go very far into the guts of each area because some of them (like APM) are going to need some extra special love. But I can certainly make the changes you want to the firehose section because it's pretty straightforward. :-) Thanks again.

**Comment by dedemorton** [comment]
We made the following decisions during the team meeting today:

For the getting started content:
* Remove topics nested under [Get started](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/observability-get-started.html).
* Consider reframing "Logs and metrics" guide as a tutorial that uses Fleet, and move it to the Infrastructure and hosts section (or remove the topic entirely). @dedemorton to check google stats to decide whether we keep this topic.
* Move Nginx to the "Infrastructure and hosts" section and rename it to begin with "Tutorial:"
* Delete the "Traces and APM" topic and redirect users to the section under APM (@colleenmcginnis which topic do we want to redirect users to? Looking at the source files, the only getting started guide I see is under "Get started")
* Figure out what's going on with the platform docs about getting started: who owns, do we still need them, do we still want to link to them, etc.

For other tutorial-style content:
* Move topic about Kubernetes under "Infra and hosts" and add "Tutorial:" to the title.
* Move topic about Java applications under "Infra and hosts" and add "Tutorial:" to the title.

**Comment by colleenmcginnis** [comment]
>Delete the "Traces and APM" topic and redirect users to the section under APM (@colleenmcginnis which topic do we want to redirect users to? Looking at the source files, the only getting started guide I see is under "Get started")

The section that duplicates that content is [Fleet-managed APM Server](https://www.elastic.co/guide/en/observability/current/_fleet_managed_apm_server.html). As a part of https://github.com/elastic/obs-docs-projects/issues/260, I'm thinking we should change the section that contains that content from "Self manage APM Server" to "Get started" or "Set up".

File: docs/en/observability/apm.asciidoc
Comment (dedemorton) [reply docs/en/observability/apm.asciidoc:5]: I'm gonna remove the short title for now and see how other reviewers respond.

File: docs/en/observability/application-and-service-monitoring.asciidoc
Comment (dedemorton) [reply docs/en/observability/application-and-service-monitoring.asciidoc:16]: Going with the option of flagging this as a tutorial and moving it to the end of the section.

**Comment by dedemorton** [comment]
@alaudazzi  Thanks for your input! Just an FYI--here's how I have addressed your feedback:

>The [Start here](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/observability-start-here.html) page is a nice entry point. How about making it lighter by removing the various “Learn about, learn how, learn more” with something like this:
New to Elastic Observability? Discover more about our Observability features and how to get started.

Nice idea. Done!

>The vicinity of Quick starts and Get started is confusing. How about promoting the Quick starts one level up and make them more visible in the TOC?

I'll look at this after the quickstarts are merged, but I think with the getting started section slimmed down, your suggestion makes sense.

>In [Applications and services](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/application-and-service-monitoring.html), it is weird to have “monitoring” for most of the TOC entries, except for APM and Synthetics.

It is. Already fixed based on Colleen's feedback.

>I love the [Cloud](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/cloud-monitoring.html) section!

Yay

>In [Monitor AWS with Elastic Agent](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/monitor-aws-elastic-agent.html) and [Monitor AWS with Amazon Data Firehose](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/ingest-aws-firehose.html), we can alleviate the TOC by removing Monitor from headers?

Done

>[Reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/reference.html) seems vague and is not adding much value to the navigation. How about moving [Fields reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/fields-reference.html) and [Elastic Entity Model](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/elastic-entity-model.html) on level up and removing [Reference](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/reference.html)?

Let's wait and get more input from reviewers on this. I think we might get some mixed opinions about this.

**Comment by dedemorton** [comment]
>The section that duplicates that content is [Fleet-managed APM Server](https://www.elastic.co/guide/en/observability/current/_fleet_managed_apm_server.html). As a part of https://github.com/elastic/obs-docs-projects/issues/260, I'm thinking we should change the section that contains that content from "Self manage APM Server" to "Get started" or "Set up".

@colleenmcginnis So the page I need to link to does not have an ID. I'm going to add an ID that you can change later, if you want.

**Comment by colleenmcginnis** [comment]
@dedemorton I ran into similar errors recently...

```
INFO:build_docs:Bad cross-document links:
--
  | INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/apm/guide/master/_fleet_managed_apm_server.html contains broken links to:
  | INFO:build_docs:   - en/observability/master/_fleet_managed_apm_server.html
  | INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/apm/guide/master/_step_2_add_and_configure_the_apm_integration.html contains broken links to:
  | INFO:build_docs:   - en/observability/master/_step_2_add_and_configure_the_apm_integration.html
  | INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/apm/guide/master/apm-quick-start.html contains broken links to:
  | INFO:build_docs:   - en/observability/master/traces-get-started.html
```

... to fix them, you'll need to update the redirects in [`docs/en/apm-server/redirects.asciidoc`](https://github.com/dedemorton/observability-docs/blob/issue%23247/docs/en/apm-server/redirects.asciidoc?plain=1#L489-L522).

**Comment by dedemorton** [comment]
@colleenmcginnis @alaudazzi @bmorelli25 @mdbirnstiehl I'm curious what you think of merging the "get started" and quickstarts into one section:

image

That way there is one page that has all the links to quickstarts and getting started.

**Comment by alaudazzi** [comment]
> I'm curious what you think of merging the "get started" and quickstarts into one section:

It makes sense, but I would remove "Quickstart" and reword the headings. It sounds weird to have a quickstart into a get started. How about this:

- Get started with Elastic Agent
   - Monitor your hosts
   - Monitor your Kubernetes cluster
- Get started with data from Splunk

**Comment by dedemorton** [comment]
>It sounds weird to have a quickstart into a get started.

Why? In my mind, a quickstart is just a type of getting started document. I also think the quickstart label makes it clear that we're offering an accelerated path. I'm curious if other writers share your concern. I want to indicate in the nav somehow that these topics are special.

**Comment by alaudazzi** [comment]
@colleenmcginnis @dedemorton 
The Elastic documentation guidelines cover these [content types](https://docs.elastic.dev/content-architecture/content-types), where `get started` and `quickstart` are not directly referenced but might correspond to `how-to` and `tutorial`.

Based on previous conversations we had at Elastic about content types and IA, I recall that it has always been hard to exactly define what is what. My understanding is that:

A `Get started` is to learn how to cook. More general. For novice users.
A `Quickstart` is to learn how to cook pasta, fish, meat, and so on. More specific. For expert users.

They are two different content types, they address two different audiences, so it would be more natural to have them at the same level.

**Comment by colleenmcginnis** [comment]
>A `Get started` is to learn how to cook. More general. For novice users.
>A `Quickstart` is to learn how to cook pasta, fish, meat, and so on. More specific. For expert users.

Interesting. Do you think [Get started with data from Splunk](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/splunk-get-started.html#splunk-step-two) is more general than [Quickstart: Monitor your Kubernetes cluster with Elastic Agent](https://observability-docs_bk_4134.docs-preview.app.elstc.co/guide/en/observability/master/monitor-k8s-logs-metrics-with-elastic-agent.html)?

**Comment by dedemorton** [comment]
> I'm curious what the distinction is between "get started" and "quickstart". 

Good question! Originally I had all the quickstarts under a "Quickstarts" section that explained what we mean by quickstarts (the term "quickstarts" does appear in the UI and I think the original issue requested that we describe what a quickstart is) Originally the titles of these quickstarts exactly matched the UI, but I was asked to make the title more goal oriented. The section that describes quickstarts is now [here](https://github.com/elastic/observability-docs/pull/4134/files#diff-288558b660bf6e2e9c36f886d36027de254feee07c9e68259651326d49f4006aR28), but maybe it's too subtle and buried in the "Get started" topic? I mainly used "quickstart" in the topic titles to map this content to what users see in the UI and also let users know that the topic contains into to get started quickly.

File: docs/en/observability/splunk-get-started.asciidoc
Comment (dedemorton) [inline docs/en/observability/splunk-get-started.asciidoc:2]:
```suggestion
= Migrate data from Splunk
```

Comment (dedemorton) [reply docs/en/observability/splunk-get-started.asciidoc:2]: @colleenmcginnis Maybe we can just sidestep the question of what we mean by getting started by renaming this topic, which is kind of an outlier. Unfortunately the topic relies on users knowing some Elastic concepts up front, but maybe we can make the topic better later on.

**Comment by dedemorton** [comment]
Met with other obs writers today. Decision was to:

* Change "Start here" to "Get started" and use the current content from the getting started topic as the main page.
* Flatten out the quickstart structure and possibly remove "Quick start" from the title. (I'll see how it looks)

**Comment by akhileshpok** [comment]
@dedemorton - I think that universal profiling should be under infrastructure. IMO, we should have either'Get started' or 'Quick start' in the nav. Please let me know if you would like to me to review any other items. Thanks

**Comment by dedemorton** [comment]
I've moved Universal Profiling to appear under the infra/hosts monitoring section.  It didn't seem right to nest Universal Profiling under a topic that describes the Infrastructure UI, so I added a new container topic with a jump table to the subsections. That's the best I can do for the content as it is today. 

image

**Comment by dedemorton** [comment]
@akhileshpok Mainly I need you to confirm that the navigation/TOC structure looks good. With the new structure, we'll eventually need to add more content and rewrite some of the existing topics, but that work is out of scope for this PR.

We'll backport this to 8.16, so the nav won't go live right away. That'll give other stakeholders an opportunity to suggest tweaks. Thanks!

**Comment by dedemorton** [comment]
I got the go-ahead from Akhilesh over slack. I just need to move the troubleshooting section, and I think we're ready to merge this into main.

Akhilesh Pokhariyal
[Yesterday at 3:26 AM](https://elastic.slack.com/archives/CJYH0RWR3/p1725272807607989?thread_ts=1724891972.954469)
[@dede](https://elastic.slack.com/team/U0JFQ58Q5)
 - I like the new IA for Obs, as it is consistent with the entity centric vision. I would suggest moving Troubleshooting out of the 'Apps and services' section and make it a top-level option. We can always improve the quality, quantity and consistency of the content under various sections, but this is a good starting point in my opinion. Thanks (edited)

**Review by alaudazzi — APPROVED** [review]
LGTM

**Review by mdbirnstiehl — APPROVED** [review]

**Pull request merged**
